tests/test_queue.py

```diff
@@ -900,6 +900,33 @@ def regular_workflow() -> None:
     assert queue_entries_are_cleaned_up(dbos)
 
 
+# Test a race condition between removing a task from the queue and flushing the status buffer
+def test_resuming_already_completed_queue_workflow(dbos: DBOS) -> None:
+    _buffer_flush_interval_secs = 999999 # Disable buffer flush
```

Comment:
Should we declare it to be `global` to change it?

Reply:
Good catch I revamped the test to disable flushing and doing it manually.